miden-lib/src/notes/mod.rs

```diff
@@ -12,7 +12,7 @@ use miden_objects::{
 };
 use utils::build_swap_tag;
 
-pub mod scripts;
```

Comment:
@PhilippGackstatter is there any reason as to why this is being changed? We want to access scripts from the outside (ie, for comparing roots) which is why it was changed to be explicitly public.

Reply:
I think I might have asked not to export things that are not needed - but I think this change we can revert. Let me push a small commit to do this.